src/main/java/Main.java

```diff
@@ -10,10 +10,11 @@
 
 public class Main {
     public static void main(String[] args) throws IOException {
-        String searchText = getSearchText();
+//        String searchText = getSearchText();
+        String searchText = "";
         searchText = searchText.isEmpty() ?
```

Comment:
Tu jakieś hacki straszne. To jest zgaduję demo? Przeniósł bym całe do getSearchText() jak już.